src/server/index.js

```diff
@@ -18,6 +18,9 @@ app.use(express.urlencoded({ extended: true }));
 const pokemonRouter = require("./routers/pokemon")
 app.use("/pokemon", pokemonRouter)
 
+const userRouter = require("./routers/user")
+app.use("/user", userRouter)
+
 
```

Comment:
When doing routing we should definitely consider changing these routers so the average user can't accidentally access them but they work for now for development

Reply:
Yes makes sense. Do you mean we should make the route dynamic depending on the user details?

Reply:
I'm not entirely sure how to approach it, if I'm honest, but I do feel like it should be changed haha. Something to research/ask teachers about I suppose